programs/associated-token-account/Create.go

```diff
@@ -174,10 +182,12 @@ func (inst *Create) EncodeToTree(parent treeout.Branches) {
 				ParentFunc(func(instructionBranch treeout.Branches) {
 
 					// Parameters of the instruction:
-					instructionBranch.Child("Params[len=0]").ParentFunc(func(paramsBranch treeout.Branches) {})
+					instructionBranch.Child("Params").ParentFunc(func(paramsBranch treeout.Branches) {
+						paramsBranch.Child(ag_format.Param("Idempotent", inst.Idempotent))
+					})
 
 					// Accounts of the instruction:
-					instructionBranch.Child("Accounts[len=7").ParentFunc(func(accountsBranch treeout.Branches) {
+					instructionBranch.Child("Accounts[len=7]").ParentFunc(func(accountsBranch treeout.Branches) {
```

Comment:
This code updates the pretty printer for this instruction to include whether the idempotent flag is set